scaling_performance/scaling_cluster_metrics.adoc

```diff
@@ -141,10 +140,45 @@ replicas, run:
 
 Alternatively, update your inventory file and re-run the
 xref:../install_config/cluster_metrics.adoc#deploying-the-metrics-components[deployment].
-endif::openshift-enterprise[]
 
 [NOTE]
 ====
 If you add a new node to or remove an existing node from a Cassandra cluster,
 the data stored in the cluster rebalances across the cluster.
 ====
+
+To scale down:
+
+. If remotely accessing the container, run the following for the Cassandra node
+you want to remove:
++
+----
+$ oc exec -it <hawkular-cassandra-pod> nodetool decommission
+----
++
+If locally accessing the container, run the following instead:
++
+----
+$ oc rsh <hawkular-cassandra-pod> nodetool decommission
+----
++
+This command can take a while to run since it copies data across the cluster.
+You can monitor the decommission progress with `nodetool netstats -H`.
+
+. Once the previous command succeeds, scale down the `rc` for the Cassandra instance to `0`.
++
+----
+# oc scale -n openshift-infra --replicas=0 rc hawkular-metrics
```

Comment:
please change to:
oc scale -n openshift-infra --replicas=0 rc <hawkular-cassandra-rc>